modules/apps/object/object-service/src/main/java/com/liferay/object/internal/item/selector/SystemObjectEntryItemSelectorView.java

```diff
@@ -225,10 +231,26 @@ public String getTitle(Locale locale) {
 				return StringPool.BLANK;
 			}
 
-			Map<String, Object> modelAttributes =
-				_baseModel.getModelAttributes();
+			User user = _userLocalService.fetchUser(
+				PrincipalThreadLocal.getUserId());
```

Comment:
I believe that we can't do that because the `getUserId()` is getting the baseModel creator's userId, but in this case we need to get the current user's id